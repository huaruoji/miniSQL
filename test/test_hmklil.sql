CREATE DATABASE db_kqspsuxy;
USE DATABASE db_kqspsuxy;
CREATE TABLE table_sqacqe (
    col_oekq FLOAT,
    col_jzhi FLOAT,
    col_zxfo FLOAT,
    col_wuoc INTEGER
);
INSERT INTO table_sqacqe VALUES (-2853.34, 7525.99, 4832.59, -3064);
INSERT INTO table_sqacqe VALUES (-2798.94, -5713.07, -4705.18, -480);
INSERT INTO table_sqacqe VALUES (-3699.72, 6145.77, 7922.71, -2232);
INSERT INTO table_sqacqe VALUES (6158.38, -3297.17, -2221.96, -6916);
INSERT INTO table_sqacqe VALUES (-6120.41, -787.47, 8283.57, 638);
INSERT INTO table_sqacqe VALUES (3097.14, 8391.11, 1571.46, 4721);
INSERT INTO table_sqacqe VALUES (265.86, 8771.25, 1047.19, 4419);
INSERT INTO table_sqacqe VALUES (1194.87, 1440.04, 6089.94, 1670);
INSERT INTO table_sqacqe VALUES (-9514.89, -6594.89, 5979.96, 8791);
INSERT INTO table_sqacqe VALUES (-8131.2, 7959.34, 7187.53, 8020);
SELECT col_wuoc FROM table_sqacqe;
SELECT col_oekq FROM table_sqacqe;
SELECT * FROM table_sqacqe;
CREATE TABLE table_wigzbm (
    col_tifo TEXT,
    col_jsrf INTEGER,
    col_ldzd INTEGER,
    col_dufm TEXT
);
INSERT INTO table_wigzbm VALUES ('fxbnzduplj', -6462, 7884, 'zpjkyhvjbc');
INSERT INTO table_wigzbm VALUES ('ismyajuvmy', -1583, 1270, 'qqvvumhlpe');
INSERT INTO table_wigzbm VALUES ('jvxvgrjmmi', -4935, -8620, 'nevoyieysk');
INSERT INTO table_wigzbm VALUES ('fforzvoite', -7288, -843, 'yupdvfinrk');
INSERT INTO table_wigzbm VALUES ('uvotbejqoz', 2435, -2997, 'xlhryjegzd');
INSERT INTO table_wigzbm VALUES ('wdilqnrhjk', 2524, 519, 'fanyphzhsa');
INSERT INTO table_wigzbm VALUES ('chgndephys', 2286, -1142, 'yxccsmznrp');
INSERT INTO table_wigzbm VALUES ('ujzgmbivpz', -181, 2742, 'cicjkzdokl');
INSERT INTO table_wigzbm VALUES ('bcqtioxusv', 5481, -9142, 'gezhxdiwcb');
INSERT INTO table_wigzbm VALUES ('zsewqjkmwn', 4445, 2478, 'yqkrfibbjq');
SELECT col_dufm FROM table_wigzbm;
SELECT col_tifo, col_ldzd FROM table_wigzbm;
SELECT col_dufm FROM table_wigzbm;
CREATE TABLE table_hhdjat (
    col_xewx FLOAT,
    col_njjz FLOAT,
    col_frgp INTEGER,
    col_snfb INTEGER,
    col_dnjr FLOAT,
    col_eupv TEXT,
    col_ktgz FLOAT
);
INSERT INTO table_hhdjat VALUES (9772.86, -9817.92, -8291, 6773, -9764.47, 'bwaawyysof', 7022.49);
INSERT INTO table_hhdjat VALUES (-7558.53, 3031.64, -165, -8267, 5455.65, 'vqrfkrrian', 6565.48);
INSERT INTO table_hhdjat VALUES (-1796.68, 7496.96, -2378, -1570, 7435.59, 'thxcuxfzca', -9128.65);
INSERT INTO table_hhdjat VALUES (5136.63, -2628.58, -3911, 2958, 9303.71, 'truywzckzf', 1322.08);
INSERT INTO table_hhdjat VALUES (1809.97, 8514.65, -5545, 6728, -5536.34, 'azfugpbxvp', 4246.49);
INSERT INTO table_hhdjat VALUES (2725.7, 9322.13, -3919, 9287, -1568.08, 'pnhtiytgvr', -7141.52);
INSERT INTO table_hhdjat VALUES (-2430.69, 8762.93, 9468, 6843, -9720.96, 'ylivqmnbjr', -2265.05);
INSERT INTO table_hhdjat VALUES (4223.78, -6493.75, 2585, 5408, -1209.08, 'fcuprjkdcy', 4723.32);
INSERT INTO table_hhdjat VALUES (8640.36, -7210.33, -8922, -4704, 6955.22, 'uxyrgjovzh', -7713.73);
INSERT INTO table_hhdjat VALUES (2094.07, -2717.4, 340, -1846, -7198.92, 'aelktyjyge', -5021.43);
SELECT * FROM table_hhdjat;
SELECT col_xewx, col_njjz, col_eupv, col_dnjr, col_frgp, col_ktgz, col_snfb FROM table_hhdjat;
SELECT col_dnjr, col_ktgz FROM table_hhdjat;
