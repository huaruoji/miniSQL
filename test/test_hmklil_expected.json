{
  "tables": {
    "table_sqacqe": {
      "columns": {
        "col_oekq": "FLOAT",
        "col_jzhi": "FLOAT",
        "col_zxfo": "FLOAT",
        "col_wuoc": "INTEGER"
      },
      "rows": [
        {
          "col_oekq": -2853.34,
          "col_jzhi": 7525.99,
          "col_zxfo": 4832.59,
          "col_wuoc": -3064
        },
        {
          "col_oekq": -2798.94,
          "col_jzhi": -5713.07,
          "col_zxfo": -4705.18,
          "col_wuoc": -480
        },
        {
          "col_oekq": -3699.72,
          "col_jzhi": 6145.77,
          "col_zxfo": 7922.71,
          "col_wuoc": -2232
        },
        {
          "col_oekq": 6158.38,
          "col_jzhi": -3297.17,
          "col_zxfo": -2221.96,
          "col_wuoc": -6916
        },
        {
          "col_oekq": -6120.41,
          "col_jzhi": -787.47,
          "col_zxfo": 8283.57,
          "col_wuoc": 638
        },
        {
          "col_oekq": 3097.14,
          "col_jzhi": 8391.11,
          "col_zxfo": 1571.46,
          "col_wuoc": 4721
        },
        {
          "col_oekq": 265.86,
          "col_jzhi": 8771.25,
          "col_zxfo": 1047.19,
          "col_wuoc": 4419
        },
        {
          "col_oekq": 1194.87,
          "col_jzhi": 1440.04,
          "col_zxfo": 6089.94,
          "col_wuoc": 1670
        },
        {
          "col_oekq": -9514.89,
          "col_jzhi": -6594.89,
          "col_zxfo": 5979.96,
          "col_wuoc": 8791
        },
        {
          "col_oekq": -8131.2,
          "col_jzhi": 7959.34,
          "col_zxfo": 7187.53,
          "col_wuoc": 8020
        }
      ]
    },
    "table_wigzbm": {
      "columns": {
        "col_tifo": "TEXT",
        "col_jsrf": "INTEGER",
        "col_ldzd": "INTEGER",
        "col_dufm": "TEXT"
      },
      "rows": [
        {
          "col_tifo": "fxbnzduplj",
          "col_jsrf": -6462,
          "col_ldzd": 7884,
          "col_dufm": "zpjkyhvjbc"
        },
        {
          "col_tifo": "ismyajuvmy",
          "col_jsrf": -1583,
          "col_ldzd": 1270,
          "col_dufm": "qqvvumhlpe"
        },
        {
          "col_tifo": "jvxvgrjmmi",
          "col_jsrf": -4935,
          "col_ldzd": -8620,
          "col_dufm": "nevoyieysk"
        },
        {
          "col_tifo": "fforzvoite",
          "col_jsrf": -7288,
          "col_ldzd": -843,
          "col_dufm": "yupdvfinrk"
        },
        {
          "col_tifo": "uvotbejqoz",
          "col_jsrf": 2435,
          "col_ldzd": -2997,
          "col_dufm": "xlhryjegzd"
        },
        {
          "col_tifo": "wdilqnrhjk",
          "col_jsrf": 2524,
          "col_ldzd": 519,
          "col_dufm": "fanyphzhsa"
        },
        {
          "col_tifo": "chgndephys",
          "col_jsrf": 2286,
          "col_ldzd": -1142,
          "col_dufm": "yxccsmznrp"
        },
        {
          "col_tifo": "ujzgmbivpz",
          "col_jsrf": -181,
          "col_ldzd": 2742,
          "col_dufm": "cicjkzdokl"
        },
        {
          "col_tifo": "bcqtioxusv",
          "col_jsrf": 5481,
          "col_ldzd": -9142,
          "col_dufm": "gezhxdiwcb"
        },
        {
          "col_tifo": "zsewqjkmwn",
          "col_jsrf": 4445,
          "col_ldzd": 2478,
          "col_dufm": "yqkrfibbjq"
        }
      ]
    },
    "table_hhdjat": {
      "columns": {
        "col_xewx": "FLOAT",
        "col_njjz": "FLOAT",
        "col_frgp": "INTEGER",
        "col_snfb": "INTEGER",
        "col_dnjr": "FLOAT",
        "col_eupv": "TEXT",
        "col_ktgz": "FLOAT"
      },
      "rows": [
        {
          "col_xewx": 9772.86,
          "col_njjz": -9817.92,
          "col_frgp": -8291,
          "col_snfb": 6773,
          "col_dnjr": -9764.47,
          "col_eupv": "bwaawyysof",
          "col_ktgz": 7022.49
        },
        {
          "col_xewx": -7558.53,
          "col_njjz": 3031.64,
          "col_frgp": -165,
          "col_snfb": -8267,
          "col_dnjr": 5455.65,
          "col_eupv": "vqrfkrrian",
          "col_ktgz": 6565.48
        },
        {
          "col_xewx": -1796.68,
          "col_njjz": 7496.96,
          "col_frgp": -2378,
          "col_snfb": -1570,
          "col_dnjr": 7435.59,
          "col_eupv": "thxcuxfzca",
          "col_ktgz": -9128.65
        },
        {
          "col_xewx": 5136.63,
          "col_njjz": -2628.58,
          "col_frgp": -3911,
          "col_snfb": 2958,
          "col_dnjr": 9303.71,
          "col_eupv": "truywzckzf",
          "col_ktgz": 1322.08
        },
        {
          "col_xewx": 1809.97,
          "col_njjz": 8514.65,
          "col_frgp": -5545,
          "col_snfb": 6728,
          "col_dnjr": -5536.34,
          "col_eupv": "azfugpbxvp",
          "col_ktgz": 4246.49
        },
        {
          "col_xewx": 2725.7,
          "col_njjz": 9322.13,
          "col_frgp": -3919,
          "col_snfb": 9287,
          "col_dnjr": -1568.08,
          "col_eupv": "pnhtiytgvr",
          "col_ktgz": -7141.52
        },
        {
          "col_xewx": -2430.69,
          "col_njjz": 8762.93,
          "col_frgp": 9468,
          "col_snfb": 6843,
          "col_dnjr": -9720.96,
          "col_eupv": "ylivqmnbjr",
          "col_ktgz": -2265.05
        },
        {
          "col_xewx": 4223.78,
          "col_njjz": -6493.75,
          "col_frgp": 2585,
          "col_snfb": 5408,
          "col_dnjr": -1209.08,
          "col_eupv": "fcuprjkdcy",
          "col_ktgz": 4723.32
        },
        {
          "col_xewx": 8640.36,
          "col_njjz": -7210.33,
          "col_frgp": -8922,
          "col_snfb": -4704,
          "col_dnjr": 6955.22,
          "col_eupv": "uxyrgjovzh",
          "col_ktgz": -7713.73
        },
        {
          "col_xewx": 2094.07,
          "col_njjz": -2717.4,
          "col_frgp": 340,
          "col_snfb": -1846,
          "col_dnjr": -7198.92,
          "col_eupv": "aelktyjyge",
          "col_ktgz": -5021.43
        }
      ]
    }
  },
  "queries": [
    {
      "query": "SELECT col_wuoc FROM table_sqacqe;",
      "result": [
        {
          "col_wuoc": -3064
        },
        {
          "col_wuoc": -480
        },
        {
          "col_wuoc": -2232
        },
        {
          "col_wuoc": -6916
        },
        {
          "col_wuoc": 638
        },
        {
          "col_wuoc": 4721
        },
        {
          "col_wuoc": 4419
        },
        {
          "col_wuoc": 1670
        },
        {
          "col_wuoc": 8791
        },
        {
          "col_wuoc": 8020
        }
      ]
    },
    {
      "query": "SELECT col_oekq FROM table_sqacqe;",
      "result": [
        {
          "col_oekq": -2853.34
        },
        {
          "col_oekq": -2798.94
        },
        {
          "col_oekq": -3699.72
        },
        {
          "col_oekq": 6158.38
        },
        {
          "col_oekq": -6120.41
        },
        {
          "col_oekq": 3097.14
        },
        {
          "col_oekq": 265.86
        },
        {
          "col_oekq": 1194.87
        },
        {
          "col_oekq": -9514.89
        },
        {
          "col_oekq": -8131.2
        }
      ]
    },
    {
      "query": "SELECT * FROM table_sqacqe;",
      "result": [
        {
          "col_oekq": -2853.34,
          "col_jzhi": 7525.99,
          "col_zxfo": 4832.59,
          "col_wuoc": -3064
        },
        {
          "col_oekq": -2798.94,
          "col_jzhi": -5713.07,
          "col_zxfo": -4705.18,
          "col_wuoc": -480
        },
        {
          "col_oekq": -3699.72,
          "col_jzhi": 6145.77,
          "col_zxfo": 7922.71,
          "col_wuoc": -2232
        },
        {
          "col_oekq": 6158.38,
          "col_jzhi": -3297.17,
          "col_zxfo": -2221.96,
          "col_wuoc": -6916
        },
        {
          "col_oekq": -6120.41,
          "col_jzhi": -787.47,
          "col_zxfo": 8283.57,
          "col_wuoc": 638
        },
        {
          "col_oekq": 3097.14,
          "col_jzhi": 8391.11,
          "col_zxfo": 1571.46,
          "col_wuoc": 4721
        },
        {
          "col_oekq": 265.86,
          "col_jzhi": 8771.25,
          "col_zxfo": 1047.19,
          "col_wuoc": 4419
        },
        {
          "col_oekq": 1194.87,
          "col_jzhi": 1440.04,
          "col_zxfo": 6089.94,
          "col_wuoc": 1670
        },
        {
          "col_oekq": -9514.89,
          "col_jzhi": -6594.89,
          "col_zxfo": 5979.96,
          "col_wuoc": 8791
        },
        {
          "col_oekq": -8131.2,
          "col_jzhi": 7959.34,
          "col_zxfo": 7187.53,
          "col_wuoc": 8020
        }
      ]
    },
    {
      "query": "SELECT col_dufm FROM table_wigzbm;",
      "result": [
        {
          "col_dufm": "zpjkyhvjbc"
        },
        {
          "col_dufm": "qqvvumhlpe"
        },
        {
          "col_dufm": "nevoyieysk"
        },
        {
          "col_dufm": "yupdvfinrk"
        },
        {
          "col_dufm": "xlhryjegzd"
        },
        {
          "col_dufm": "fanyphzhsa"
        },
        {
          "col_dufm": "yxccsmznrp"
        },
        {
          "col_dufm": "cicjkzdokl"
        },
        {
          "col_dufm": "gezhxdiwcb"
        },
        {
          "col_dufm": "yqkrfibbjq"
        }
      ]
    },
    {
      "query": "SELECT col_tifo, col_ldzd FROM table_wigzbm;",
      "result": [
        {
          "col_tifo": "fxbnzduplj",
          "col_ldzd": 7884
        },
        {
          "col_tifo": "ismyajuvmy",
          "col_ldzd": 1270
        },
        {
          "col_tifo": "jvxvgrjmmi",
          "col_ldzd": -8620
        },
        {
          "col_tifo": "fforzvoite",
          "col_ldzd": -843
        },
        {
          "col_tifo": "uvotbejqoz",
          "col_ldzd": -2997
        },
        {
          "col_tifo": "wdilqnrhjk",
          "col_ldzd": 519
        },
        {
          "col_tifo": "chgndephys",
          "col_ldzd": -1142
        },
        {
          "col_tifo": "ujzgmbivpz",
          "col_ldzd": 2742
        },
        {
          "col_tifo": "bcqtioxusv",
          "col_ldzd": -9142
        },
        {
          "col_tifo": "zsewqjkmwn",
          "col_ldzd": 2478
        }
      ]
    },
    {
      "query": "SELECT col_dufm FROM table_wigzbm;",
      "result": [
        {
          "col_dufm": "zpjkyhvjbc"
        },
        {
          "col_dufm": "qqvvumhlpe"
        },
        {
          "col_dufm": "nevoyieysk"
        },
        {
          "col_dufm": "yupdvfinrk"
        },
        {
          "col_dufm": "xlhryjegzd"
        },
        {
          "col_dufm": "fanyphzhsa"
        },
        {
          "col_dufm": "yxccsmznrp"
        },
        {
          "col_dufm": "cicjkzdokl"
        },
        {
          "col_dufm": "gezhxdiwcb"
        },
        {
          "col_dufm": "yqkrfibbjq"
        }
      ]
    },
    {
      "query": "SELECT * FROM table_hhdjat;",
      "result": [
        {
          "col_xewx": 9772.86,
          "col_njjz": -9817.92,
          "col_frgp": -8291,
          "col_snfb": 6773,
          "col_dnjr": -9764.47,
          "col_eupv": "bwaawyysof",
          "col_ktgz": 7022.49
        },
        {
          "col_xewx": -7558.53,
          "col_njjz": 3031.64,
          "col_frgp": -165,
          "col_snfb": -8267,
          "col_dnjr": 5455.65,
          "col_eupv": "vqrfkrrian",
          "col_ktgz": 6565.48
        },
        {
          "col_xewx": -1796.68,
          "col_njjz": 7496.96,
          "col_frgp": -2378,
          "col_snfb": -1570,
          "col_dnjr": 7435.59,
          "col_eupv": "thxcuxfzca",
          "col_ktgz": -9128.65
        },
        {
          "col_xewx": 5136.63,
          "col_njjz": -2628.58,
          "col_frgp": -3911,
          "col_snfb": 2958,
          "col_dnjr": 9303.71,
          "col_eupv": "truywzckzf",
          "col_ktgz": 1322.08
        },
        {
          "col_xewx": 1809.97,
          "col_njjz": 8514.65,
          "col_frgp": -5545,
          "col_snfb": 6728,
          "col_dnjr": -5536.34,
          "col_eupv": "azfugpbxvp",
          "col_ktgz": 4246.49
        },
        {
          "col_xewx": 2725.7,
          "col_njjz": 9322.13,
          "col_frgp": -3919,
          "col_snfb": 9287,
          "col_dnjr": -1568.08,
          "col_eupv": "pnhtiytgvr",
          "col_ktgz": -7141.52
        },
        {
          "col_xewx": -2430.69,
          "col_njjz": 8762.93,
          "col_frgp": 9468,
          "col_snfb": 6843,
          "col_dnjr": -9720.96,
          "col_eupv": "ylivqmnbjr",
          "col_ktgz": -2265.05
        },
        {
          "col_xewx": 4223.78,
          "col_njjz": -6493.75,
          "col_frgp": 2585,
          "col_snfb": 5408,
          "col_dnjr": -1209.08,
          "col_eupv": "fcuprjkdcy",
          "col_ktgz": 4723.32
        },
        {
          "col_xewx": 8640.36,
          "col_njjz": -7210.33,
          "col_frgp": -8922,
          "col_snfb": -4704,
          "col_dnjr": 6955.22,
          "col_eupv": "uxyrgjovzh",
          "col_ktgz": -7713.73
        },
        {
          "col_xewx": 2094.07,
          "col_njjz": -2717.4,
          "col_frgp": 340,
          "col_snfb": -1846,
          "col_dnjr": -7198.92,
          "col_eupv": "aelktyjyge",
          "col_ktgz": -5021.43
        }
      ]
    },
    {
      "query": "SELECT col_xewx, col_njjz, col_eupv, col_dnjr, col_frgp, col_ktgz, col_snfb FROM table_hhdjat;",
      "result": [
        {
          "col_xewx": 9772.86,
          "col_njjz": -9817.92,
          "col_eupv": "bwaawyysof",
          "col_dnjr": -9764.47,
          "col_frgp": -8291,
          "col_ktgz": 7022.49,
          "col_snfb": 6773
        },
        {
          "col_xewx": -7558.53,
          "col_njjz": 3031.64,
          "col_eupv": "vqrfkrrian",
          "col_dnjr": 5455.65,
          "col_frgp": -165,
          "col_ktgz": 6565.48,
          "col_snfb": -8267
        },
        {
          "col_xewx": -1796.68,
          "col_njjz": 7496.96,
          "col_eupv": "thxcuxfzca",
          "col_dnjr": 7435.59,
          "col_frgp": -2378,
          "col_ktgz": -9128.65,
          "col_snfb": -1570
        },
        {
          "col_xewx": 5136.63,
          "col_njjz": -2628.58,
          "col_eupv": "truywzckzf",
          "col_dnjr": 9303.71,
          "col_frgp": -3911,
          "col_ktgz": 1322.08,
          "col_snfb": 2958
        },
        {
          "col_xewx": 1809.97,
          "col_njjz": 8514.65,
          "col_eupv": "azfugpbxvp",
          "col_dnjr": -5536.34,
          "col_frgp": -5545,
          "col_ktgz": 4246.49,
          "col_snfb": 6728
        },
        {
          "col_xewx": 2725.7,
          "col_njjz": 9322.13,
          "col_eupv": "pnhtiytgvr",
          "col_dnjr": -1568.08,
          "col_frgp": -3919,
          "col_ktgz": -7141.52,
          "col_snfb": 9287
        },
        {
          "col_xewx": -2430.69,
          "col_njjz": 8762.93,
          "col_eupv": "ylivqmnbjr",
          "col_dnjr": -9720.96,
          "col_frgp": 9468,
          "col_ktgz": -2265.05,
          "col_snfb": 6843
        },
        {
          "col_xewx": 4223.78,
          "col_njjz": -6493.75,
          "col_eupv": "fcuprjkdcy",
          "col_dnjr": -1209.08,
          "col_frgp": 2585,
          "col_ktgz": 4723.32,
          "col_snfb": 5408
        },
        {
          "col_xewx": 8640.36,
          "col_njjz": -7210.33,
          "col_eupv": "uxyrgjovzh",
          "col_dnjr": 6955.22,
          "col_frgp": -8922,
          "col_ktgz": -7713.73,
          "col_snfb": -4704
        },
        {
          "col_xewx": 2094.07,
          "col_njjz": -2717.4,
          "col_eupv": "aelktyjyge",
          "col_dnjr": -7198.92,
          "col_frgp": 340,
          "col_ktgz": -5021.43,
          "col_snfb": -1846
        }
      ]
    },
    {
      "query": "SELECT col_dnjr, col_ktgz FROM table_hhdjat;",
      "result": [
        {
          "col_dnjr": -9764.47,
          "col_ktgz": 7022.49
        },
        {
          "col_dnjr": 5455.65,
          "col_ktgz": 6565.48
        },
        {
          "col_dnjr": 7435.59,
          "col_ktgz": -9128.65
        },
        {
          "col_dnjr": 9303.71,
          "col_ktgz": 1322.08
        },
        {
          "col_dnjr": -5536.34,
          "col_ktgz": 4246.49
        },
        {
          "col_dnjr": -1568.08,
          "col_ktgz": -7141.52
        },
        {
          "col_dnjr": -9720.96,
          "col_ktgz": -2265.05
        },
        {
          "col_dnjr": -1209.08,
          "col_ktgz": 4723.32
        },
        {
          "col_dnjr": 6955.22,
          "col_ktgz": -7713.73
        },
        {
          "col_dnjr": -7198.92,
          "col_ktgz": -5021.43
        }
      ]
    }
  ],
  "database": "db_kqspsuxy"
}